clang/lib/Driver/ToolChain.cpp

```diff
@@ -1097,7 +1097,8 @@ void ToolChain::AddHIPIncludeArgs(const ArgList &DriverArgs,
                                   ArgStringList &CC1Args) const {}
 
 llvm::SmallVector<ToolChain::BitCodeLibraryInfo, 12>
-ToolChain::getHIPDeviceLibs(const ArgList &DriverArgs) const {
+ToolChain::getHIPDeviceLibs(const ArgList &DriverArgs,
+                            const Action::OffloadKind OffloadKind) const {
```

Comment:
To be consistent:
```suggestion
                            const Action::OffloadKind DeviceOffloadingKind) const {
```
or to make it more explicit that the parameter is unused:
```suggestion
                            const Action::OffloadKind) const {
```

Reply:
I'll go with the parameter rename, as it's more consistent, since the first parameter is also unused but still named. And some of the method above also have named unusued parameters.